### DISCUSSION TITLE
Running SGW

### DISCUSSION THREAD

**Discussion by ali64mohammad6464** [magma/magma#15283] [Q&A] [answered]
Hi
i want to run only SGW that compatible with other LTE components as standard interfaces.
but as i see magma has a AGW that is MME,SGW and PGW all in one.
is there any way to config magma to run only SGW?
if not, Are there sgw functions code in the Magma source code? if there is, can give me guide to how to use them and which files have sgw functionality?

**Comment by panyogesh** [comment]
As of now SGW is can't be instantiated as a separate module. It depends on mme. Code is in lte/gateway/c/core/oai/tasks/spgw.

**Reply by panyogesh** [reply, marked as answer]
Closing the isue.
